kdoctor/build.gradle.kts

```diff
@@ -8,24 +8,12 @@ repositories {
     mavenCentral()
 }
 
-//by default a linker tries to find libcurl in '/opt/homebrew/opt/curl/lib'
-//(https://github.com/ktorio/ktor/blob/main/ktor-client/ktor-client-curl/desktop/interop/libcurl.def#L5)
-//it is right for a brew build because brew knows about the curl dependency and builds for one architecture
-//(https://github.com/Homebrew/homebrew-core/blob/master/Formula/kdoctor.rb#L27)
-//but for an independent binary distribution it's better to have dependency on system universal dylib located in '/usr/lib/'
-//To use during build run './gradlew assembleReleaseExecutableMacos -PlinkWithSystemLibcurl'
-//To check it run 'otool -L ./kdoctor/build/kdoctor'
-val linkWithSystemLibcurl = hasProperty("linkWithSystemLibcurl")
```

Comment:
There's also mention of this property in the readme.

Reply:
thank you, I'll fix it